docs/targeting.md

```diff
@@ -99,4 +99,4 @@ There are three folds:
 - [Expired Cache Context Cleanup](https://github.com/DataDog/chaos-controller/blob/adb8070c989a6c25195354e3bcef3f2c839ef032/controllers/cache_handler.go#L571): on every reconcile, asserts for every existing cache the corresponding disruption exists. Also, asserts no cache listed in the map has with a already-canceled context - in which case it's deleted.
 - [Cache Deletion Safety](https://github.com/DataDog/chaos-controller/blob/adb8070c989a6c25195354e3bcef3f2c839ef032/controllers/cache_handler.go#L596): the first two methods are useful and will prevent most cases from going wrong, but still rely on the controller looping after the disruption is over to make sure the resources are released. Started in its own goroutine this function will, every minute, poll the disruption. If it is not found the cache context will be canceled, releasing the resources it used. If the controller reconciles again, the second listed function will erase the cache from the map, as the context will be found canceled.
 
-To ensure no cache is running loose, a gauge indicating the amount of current cache entries is set to be monitored; it can be substracted to the gauge of live disruptions to alert in case of sustained negative results.
+To ensure no cache is running loose, a gauge indicating the amount of current cache entries is set to be monitored; it can be substracted from the gauge of live disruptions to alert in case of sustained negative results.
```

Comment:
```suggestion
To ensure no cache is running loose, a gauge indicating the amount of current cache entries is set to be monitored; it can be subtracted from the gauge of live disruptions to alert in case of sustained negative results.
```